app-requests.http

```diff
@@ -1,27 +1,38 @@
 ### GET generic request without params:
 GET http://localhost:4000/meteors
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with all set of parameters:
 GET http://localhost:4000/meteors?count=true&start_date=2024-03-04&end_date=2024-03-08&were_dangerous_meteors=true
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with dangerous parameter only:
 GET http://localhost:4000/meteors?were_dangerous_meteors=true
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with range of dates and dangerous paramters
 GET http://localhost:4000/meteors?were_dangerous_meteors=true&start_date=2024-03-04&end_date=2024-03-08
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with range of dates parametr only:
 GET http://localhost:4000/meteors?start_date=2024-03-04&end_date=2024-03-08
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with 'count' parametr only:
 GET http://localhost:4000/meteors?count=true
-Content-Type: "application/json"
+Content-Type: application/json
 
 ###GET with date and 'count' parametr:
 GET http://localhost:4000/meteors?start_date=2024-03-04&end_date=2024-03-08&count=true
-Content-Type: "application/json"
\ No newline at end of file
+Content-Type: application/json
+
+###POST obtain photo
+POST http://localhost:4000/yep-its-post-but-get-me-photo
+Content-Type: application/json
+
+{
+"id": 1234,
+"name": "Especto Patronum!",
+"user_api_key": "",
+"earth_date": "2015-6-3"
```

Comment:
As described in task, we should return most recent photo, so you can remove `earth_date` parameter and resolve recent photo on server side

Reply:
Thanx, redone.